package.json

```diff
@@ -18,7 +18,7 @@
     "iot"
   ],
   "author": "",
-  "license": "MPL2",
+  "license": "MPL-2.0",
```

Comment:
Finally decided to get rid of this annoying npm complaint about invalid license format :)